libyul/optimiser/LoadResolver.cpp

```diff
@@ -30,8 +30,6 @@
 #include <libyul/AST.h>
 #include <libyul/Utilities.h>
 
-#include <libevmasm/Instruction.h>
-#include <libevmasm/GasMeter.h>
```

Comment:
This file does use `GasMeter` though. It must be included through some other header but I think it should stay here too.

Reply:
Good catch; fixed.